mmv1/products/healthcare/terraform.yaml

```diff
@@ -82,6 +82,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           dataset_name: "example-dataset"
           dicom_store_name: "example-dicom-store"
           pubsub_topic: "dicom-notifications"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "healthcare_dicom_store_bq_stream"
+        min_version: beta
+        primary_resource_id: "default"
+        vars:
+          dataset_name: "example-dataset"
```

Comment:
Tests are failing with:

```
Error: Error creating Dataset: googleapi: Error 400: Invalid dataset ID "tf-test-example-bq-dswdjrdkow69". Dataset IDs must be alphanumeric (plus underscores) and must be at most 1024 characters long., invalid

on terraform_plugin_test.tf line 36, in resource "google_bigquery_dataset" "bq_dataset":
36: resource "google_bigquery_dataset" "bq_dataset" {
```

It sounds like hyphens may not be allowed. Perhaps an underscore would work?